### DISCUSSION TITLE
Is it possible to use Impala to query the cdc data in arctic？

### DISCUSSION THREAD

**Discussion by Cqz666** [apache/amoro#1110] [Q&A]
What I know so far is that impala cannot query the native iceberg v2 format.
So If I use arctic's mixed hive format, can I query the change data through Impala?

**Comment by zhoujinsong** [comment]
Hi, We have implemented an Impala connector for Arctic. But it hasn't been open-sourced yet.
If you need it, I can give you a compiled plugin, and we'll consider open-sourcing this part of the implementation if more users need it in the future.
